--- lib/authorization/aasm_roles.rb	2014-03-12 23:33:33.000000000 -0300
+++ lib/authorization/aasm_roles.rb	2014-03-12 23:32:39.000000000 -0300
@@ -10,30 +10,30 @@
         include StatefulRolesInstanceMethods
         include AASM
         aasm_column :state
-        aasm_initial_state :pending
-        aasm_state :passive
-        aasm_state :pending, :enter => :make_activation_code
-        aasm_state :active,  :enter => :do_activate
-        aasm_state :suspended
-        aasm_state :deleted, :enter => :do_delete
+        aasm.initial_state :pending
+        aasm.state :passive
+        aasm.state :pending, :enter => :make_activation_code
+        aasm.state :active,  :enter => :do_activate
+        aasm.state :suspended
+        aasm.state :deleted, :enter => :do_delete
 
-        aasm_event :register do
+        aasm.event :register do
           transitions :from => :passive, :to => :pending, :guard => Proc.new {|u| !(u.crypted_password.blank? && u.password.blank?) }
         end
         
-        aasm_event :activate do
+        aasm.event :activate do
           transitions :from => :pending, :to => :active 
         end
         
-        aasm_event :suspend do
+        aasm.event :suspend do
           transitions :from => [:passive, :pending, :active], :to => :suspended
         end
         
-        aasm_event :delete do
+        aasm.event :delete do
           transitions :from => [:passive, :pending, :active, :suspended], :to => :deleted
         end
 
-        aasm_event :unsuspend do
+        aasm.event :unsuspend do
           transitions :from => :suspended, :to => :active,  :guard => Proc.new {|u| !u.activated_at.blank? }
           transitions :from => :suspended, :to => :pending, :guard => Proc.new {|u| !u.activation_code.blank? }
           transitions :from => :suspended, :to => :passive
